### PR TITLE
 Adds davids and Dave, roidstation's mascot. 

### PR DESCRIPTION
**Pull request by jknpj**
image

Does not actually adds them to the asteroid, yet. Davids are the offspring of goliaths, smaller, weaker but faster than their parents and somehow unable to use their tentacles as weapon. Xenobiology will most likely be able to get them before roidstation.

Dave is docile, yet weaker, david that was rescued by the engineering crew that built roidstation. He is blacklisted and can't be summoned through xenobiology.

Sprites by @Bilgecrank

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs.dm [hunk 1]
```diff
@@ -535,6 +535,44 @@ obj/item/asteroid/basilisk_hide/New()
 			C.icon_state = "[initial(C.icon_state)]_goliath[C.hidecount]"
 			C._color = "mining_goliath[C.hidecount]"
 
+/mob/living/simple_animal/hostile/asteroid/goliath/david
+	name = "david"
+	desc = "Goliaths are fiercely protective of their young, threatening them is not advised under any conditions."
```
Comment (Kurfursten) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: You could add some basic mob AI to attract adult goliaths when the young are attacked. Even some kind of enrage effect.

Comment (Exxion) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs.dm]: Side note: Comma splice here

**Comment by SGuivenne** [comment]
Do they drop the same plating that goliaths do?  If so this has problematic potential and also encourages you to immediately murder the asteroid pet.

**Comment by jknpj** [comment]
You can't butcher dave(even after using an advanced lazarus injector), you can butcher wild goliaths, however.

**Pull request merged**